docs/configuration/legend.md

```diff
@@ -163,23 +163,3 @@ var chart = new Chart(ctx, {
 ```
 
 Now when you click the legend in this chart, the visibility of the first two datasets will be linked together.
-
-## HTML Legends
```

Comment:
It may be helpful to leave this section to let users know if the possibility of creating their own legend, but remove any reference to `generateLegend()`. Possibly you could show an example of creating a simple legend